api/app/signals/apps/api/v1/serializers/expression.py

```diff
@@ -0,0 +1,47 @@
+from datapunt_api.serializers import HALSerializer
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+from signals.apps.signals.models import Expression, ExpressionContext, ExpressionType
+
+
+class ExpressionContextSerializer(serializers.ModelSerializer):
+    type = serializers.SerializerMethodField('get_type_name')
```

Comment:
This can also be done the following way: `type = serializers.CharField(source='_type__name')`
When using this there is no need for the `get_type_name` function, also if this field needs to be read-only simply add `read_only=True`

Reply:
check